td-agent/yum/td-agent.spec.in

```diff
@@ -102,7 +102,7 @@ fi
 %files
 %defattr(-,root,root,-)
 /opt/*
-%{_sysconfdir}/logrotate.d/@PACKAGE@.logrotate
+%{_sysconfdir}/logrotate.d/@PACKAGE@
```

Comment:
The below entry should handle logrotate settings:
https://github.com/clear-code/td-agent-builder/pull/85/files#diff-1bbb40b55cf61387ad828ebaa0b0e41eR116

Reply:
Thanks, I've removed the line.